src/calendar/view/CalendarAgendaView.ts

```diff
@@ -249,4 +220,21 @@ export class CalendarAgendaView implements Component<CalendarAgendaViewAttrs> {
 					}),
 			  )
 	}
+
+	private earliestEventToShowTimeIndicator(events: readonly CalendarEvent[], date: Date): number | null {
```

Comment:
you didn't have to delete a test, I thought you would make it a freestanding function but it's also okay, it's not super complicated

Reply:
Where would the test go?

Reply:
wherever the function is. You can keep it in agenda view and add a test for agenda view or you can shove it in calendar utils but it's probably not needed there.